File: solutions/1635-number-of-good-pairs/solution.c
int numIdenticalPairs(int* nums, int numsSize) {
    int n=numsSize,count=0;
    for(int i=0;i<n;i++)
        for(int j=i+1;j<n;j++)
            if(nums[i]==nums[j])
                count+=1;
    return count;
}

File: solutions/1635-number-of-good-pairs/solution.py
class Solution(object):
    def numIdenticalPairs(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        d={}
        count=0
        for i in nums:
            if i in d:
                d[i]+=1
            else:
                d[i]=1
        for i in d.values():
            if i!=1:
                count+=(i*(i-1)/2)
        return count;
